Automattic/wp-calypso
# Hosting Command Palette: Add not found tracks event

- Related to https://github.com/Automattic/dotcom-forge/issues/4670

## Proposed Changes

* Add not found event when the command palette returns zero results and there is a search query.
* Event example `calypso_hosting_command_palette_not_found`
  * `current_route : "/sites"`
  * `search_text : "my cool command"`

## Testing Instructions



* Enable event logs in Chrome dev tools: `localStorage.setItem( 'debug', 'calypso:analytics*' );` or use the vigilante chrome extension
* Open the command palette by pressing `cmd+k`.
* Enter a search text that returns a not found, for example: "My cool command"
* Observe the event `calypso_hosting_command_palette_not_found` is fired with the correct properties.
* Select a command like "Open site dashboard"
* Enter a text that returns zero sites
* Observe that we don't fire any not found event

## Screencast




https://github.com/Automattic/wp-calypso/assets/779993/25712aa5-6535-4655-902e-232bd783d5bf




## Pre-merge Checklist
